src/Geomap.js

```diff
@@ -58,8 +58,13 @@ export default class Geomap extends Viz {
 
     this._shape = constant("Circle");
     this._shapeConfig = assign(this._shapeConfig, {
+      ariaLabel: (d, i) => `${this._drawLabel(d, i)}, ${this._pointSize(d, i)}`,
       hoverOpacity: 1,
       Path: {
+        ariaLabel: (d, i) => {
+          const validColorScale = this._colorScale ? `, ${this._colorScale(d, i)}` : "";
+          return `${this._drawLabel(d, i)}, ${validColorScale}.`;
```

Comment:
remove `", "` from the return statement, as it is being added by the validColorScale.

for example, if a shape named "Alpha" does not have a validColorScale, the label is rendered as `"Alpha, ."`